src/main/java/net/sf/jabref/gui/keyboard/KeyBindingWrapper.java

```diff
@@ -0,0 +1,50 @@
+package net.sf.jabref.gui.keyboard;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import org.apache.commons.lang.WordUtils;
+
+public class KeyBindingWrapper {
```

Comment:
Rename `Wrapper` -> `ViewModel` (wrapping a datamodel for display is exactly the purpose of a view model)